### DISCUSSION TITLE
404 Page

### DISCUSSION THREAD

**Discussion by AbdelrahmanAttia20** [Security-Onion-Solutions/securityonion#7609] [Unsupported Versions] [answered]
Hello 
I have 2 questions, first one is that can I custom 404 page 

<img width="937" alt="Screen Shot 2022-03-19 at 7 03 36 AM" src="https://user-images.githubusercontent.com/61397505/159107666-77a3100b-be64-42ef-949a-24a7e1d31061.png">

I would like to have a different style of this page, so how I can do this ,or edit on it

the second Q is how I can custom overview page, what I mean of custom that to add som charts on it and some overview data on it 
I saw page [Custom your environment](https://docs.securityonion.net/en/16.04/customizing.html) and didn't find the answer to my questions

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/16.04/customizing.html is for the old Security Onion 16.04 that is no longer supported.

Make sure you are using the new Security Onion 2.3 and its documentation can be found at:
https://docs.securityonion.net/en/2.3/customizing.html
